lib/capistrano/gitflow/helpers/helper.rb

```diff
@@ -216,5 +216,26 @@ def gitflow_tag_production
       set :branch, new_production_tag
     end
 
+    def gitflow_cleanup_tags
+      return if fetch(:gitflow_keep_tags).nil?
+      tags = `git log --tags  --pretty="format:%at %D" | grep 'tag:' |sort -n | awk '{$1=""; print $0}' | tr "," "\n"| sed 's/tag:*//' | sed -e 's/^[ \t]*//'`
+      tags = tags.split.reject{|tag| tag.blank?  }
```

Comment:
I think there's a bug here. When I run `cap staging gitflow:cleanup_tags`, Cap task doesn't load Rails, so I got
`NoMethodError: undefined method `blank?' for "staging-<tag>":String`

I need to add this hack
```
class String
  def blank?
    respond_to?(:empty?) ? !!empty? : !self
  end
end
```

to make it work. Can you help fixing it?